utils/ansible_playbook_to_role.py

```diff
@@ -211,7 +211,7 @@ def added_variables(self):
             elif isinstance(task["when"], str):
                 task["when"] = [task["when"]]
 
-            variables_to_add = {tag for tag in task["tags"] if self._tag_is_valid_variable(tag)}
+            variables_to_add = {self._sanitize_tag(tag) for tag in task["tags"] if self._tag_is_valid_variable(tag)}
```

Comment:
PEP8 is complaining about this line. We may want to rewrite this for clarity.

Reply:
I know, it's about the line length. But the problem was there before and many other lines go beyong the 99 maximum line length. I refactored it to less than 120 chars, but 99 seems too little.

Reply:
Ok, I have broken some lines and it seems the issue is gone.